src/icons/visyn-action-icon.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { ActionIcon, ActionIconProps, PolymorphicComponentProps } from '@mantine/core';
+
+import { resolveIconFactory } from './adapters/util';
+import { AgnosticIconDefinition } from './types';
+
+type VisynActionIconProps = PolymorphicComponentProps<'button', ActionIconProps> & {
+  icon: AgnosticIconDefinition;
+};
+
+/**
+ * A wrapper around Mantine's ActionIcon that automatically
+ * passes the size prop to the child icon.
+ */
+export function VisynActionIcon({ size = 'md', variant = 'transparent', color = 'dark', icon, ...rest }: VisynActionIconProps) {
```

Comment:
```suggestion
export function VisynActionIcon({ size = 'md', variant = 'subtle', color = 'dark', icon, ...rest }: VisynActionIconProps) {
```

The default should be subtle to provide some feedback for the user.